ui/src/app/utils/index.tsx

```diff
@@ -536,30 +536,33 @@ export function validateInputForMySQL(
     // unclosed quote
     inputErrors.add('There is an unclosed " in the search string');
   }
+  const termsArray = searchString.trim().split(' ');
   // check each endsWith term - terms that start with *
-  searchString
-    .trim()
-    .split(' ')
-    .forEach((word) => {
-      // consecutive special chars
-      if (word.match(/[+*-]{2,}/)) {
-        inputErrors.add(
-          `Search term [${word}] cannot contain consecutive special characters `
-        );
-      }
-      if (word.match(/^\*.*\*$/)) {
-        inputErrors.add(
-          `Search term [${word}] cannot start and end in wild character '*'`
-        );
-      }
-      // length of every word without the special chars mut be >= searchTrigger
-      // for hyphenated word there will be at least 2 letters, if hyphen is removed
-      if (word.replace(/[+*"'-]/g, '').length < searchTrigger) {
-        inputErrors.add(
-          `Search term [${word}] length must be at least ${searchTrigger} characters without special characters`
-        );
-      }
-    });
+  termsArray.forEach((word) => {
+    // consecutive special chars
+    if (word.match(/[+*-]{2,}/)) {
+      inputErrors.add(
+        `Search term [${word}] cannot contain consecutive special characters `
+      );
+    }
+    if (word.match(/^\*.*\*$/)) {
+      inputErrors.add(
+        `Search term [${word}] cannot start and end in wild character '*'`
+      );
+    }
+    // length of every word without the special chars mut be >= searchTrigger
+    // for hyphenated word there will be at least 2 letters, if hyphen is removed
+    if (word.replace(/[+*"'-]/g, '').length < searchTrigger) {
+      inputErrors.add(
+        `Search term [${word}] length must be at least ${searchTrigger} characters without special characters`
+      );
+    }
+  });
+  if (termsArray.length === 1 && termsArray[0].match(/^-/)) {
+    inputErrors.add(
+      "The minus character '-' cannot be used with a single search term"
```

Comment:
@chenchals @freemabd Not sure about the wording of the error message here. Let me know if something else would make more sense.

Reply:
suggest the wording to be as below:
`Search string [${searchString}] with one term cannot start with a minus '-' character`

Reply:
@dolbeew for consistency I would use:

Search term [${searchString}] with one word cannot start with a minus character '-'

Reply:
Done. Updated the screenshot in the description.